pyproject.toml

```diff
@@ -84,4 +88,5 @@ module = [
     "faiss.*",
     "openai.*",
     "sentence_transformers.*",
+    "moviepy.*",
```

Comment:
if it's not a big requirement, we can just add it to the main dependencies. See my comment below, not sure if it'll fail anyways when it tries to import moviepy

Reply:
good idea, love it, done
below are the required dependencies of `moviepy`, it's not huge:
```
    "decorator>=4.0.2,<6.0",
    "imageio>=2.5,<3.0",
    "imageio_ffmpeg>=0.2.0",
    "numpy>=1.17.3",
    "proglog<=1.0.0",
```